src/storage/db.cpp

```diff
@@ -343,6 +379,41 @@ bool DBStore::WritePosImpl(const string& column, const K& key, const H& h, const
     return db_->Put(WriteOptions(), handleMap_.at(column), keySlice, valueSlice).ok();
 }
 
+bool DBStore::DeleteColumn(const std::string& column) {
```

Comment:
I suggest to check out `WalletStore::ClearOldData` and move it to the class `RocksDB`